Week3/MAKEME.md

```diff
@@ -54,59 +73,126 @@ let favoriteAnimals = ['blowfish', 'capricorn', 'giraffe'];
 
-3. Make a function (or functions) that generate a `ul` with `li` elements for each book ID in the array using a for loop. 
+3. Create an object and a function that takes the object as a parameter and prints out all of its names and values.
 
-4. Make an object containing information for each book. Each item (object) in this object should have the book ID you thought up in point 1 as a key, and it should have at least the following fields: title, language and author. 
+4. Create a function named `vehicleType` that receives a color, and a code, 1 for car, 2 for motorbike. And prints "a blue motorbike" for example when called as `vehicleType("blue", 2)`
 
-5. Now change the function you used to display the book ID's in a list to take the actual information about the book from the object and display that. Make sure you choose the right html elements for each piece of info, for instance, a heading for the title.
+5. Can you write the following without the `if` statement, but with just as a single line with `console.log(...);`?
 
-6. Beautify your html page with css, add sources and alts to each of the images.
- 
-7. Download book covers for each book, construct a new Object which has as keys the bookId's again, and as value the path to the image source (e.g. `{"harry_potter_blabla": "./img/harry_potter_blabla.jpg", ...}`). Now loop over these entries (_hint: `Object.keys(objectName)` gives you an array containing the keys_). Then write a function which places an image at the corresponding `li` element. Remember that Objects are not ordered, so you cannot guarantee that the first key is the first `li` element. (_Hint: you could give each `li` item an `id` tag by modifying the function you made before_)
+    ```js
+    if (3 === 3) {
+        console.log("true")
+    } else {
+        console.log("false")
+    }
+    ```
 
-```
-How to hand in your homework:
-• Upload your homework in your "hyf-javascript1" Github repository. Make sure to create a new folder "week3" first. 
-• Upload your homework files inside the week3 folder and write a description for this “commit”.
-• Your hyf-javascript1/week3 should now contain an index.html, main.css and a script.js file (and the images folder)
-• Place the link to your repository folder in Trello.
-```
+6. Create a function called `vehicle`, like before, but takes another parameter called age, so that `vehicle("blue", 1, 5)` prints "a blue used car"
 
-## Step 5: **FreeCodeCamp challenges:**
+7. Make a list of vehicles, you can add `"motorbike"`, `"caravan"`, `"bike"`, or more.
 
-_Deadline Saturday_
+8. How do you get the third element from that list?
+
+9. Change the function `vehicle` to use the list of question 7. So that `vehicle("green", 3, 1)` prints "a green new caravan".
+
+10. Use the list of vehicles to write an advertisement. So that it prints something like: `"Amazing Joe's Garage, we service cars, motorbikes, caravans and bikes."`. (Hint: use a `for` loop.)
```

Comment:
Previous classes found this assignment difficult or failed to read it carefully and implement it precisely. In my view the output should be correct English with all the punctuation in place (that's the challenge). So plurals for the vehicle types, commas followed by a single space, the word `and` to replace the final comma and closed off by a period.

Reply:
i've put your text in as a hint with the question